src/pages/ScreensPage/ScreensPage.jsx

```diff
@@ -26,8 +26,8 @@ const ScreensPage = () => {
         <Icon id="filter" size={16} />
         <SFilter_p>Filters</SFilter_p>
       </SFilterWrapper_button>
-      {!boardName && <NothinFound />}
-      {boardName && <MainDashboard />}
+      {false && <NothinFound />}
```

Comment:
i've done it only for visibility of columns with cards / needs to change by logic of boards or smth else